src/rust/src/x509/verify.rs

```
@@ -433,6 +463,9 @@ fn build_subject<'a>(
 
             Ok(Subject::IP(ip_addr))
         }
+        SubjectOwner::None => Err(pyo3::exceptions::PyRuntimeError::new_err(
+            "subject must be set for ServerVerifiers",
+        )),
```

Hmm, this should be unreachable from the Python API, right?

---

If yes, this can just be a panic.

---

Take a look and tell me if u prefer it that way. If not I can revert back to old behaviour with a panic instead of returning pyerror.